core/util/get_datasets.py

```diff
@@ -29,6 +18,80 @@ def get_one_park_dataset(
     park = park.drop(drop_columns, axis=1)
     x, y = split_sequences(park.to_numpy(), park.to_numpy(), lookback, lookahead)
 
+    # Initialise empty arrays
+    x_train, x_val, x_test = [], [], []
+    y_train, y_val, y_test = [], [], []
+
+    # Check if the data should be cross validation
+    if folds is not None:
+        x_train, y_train, x_val, y_val, x_test, y_test = get_one_cross_park(x, y, folds)
+    else:
+        x_train, y_train, x_val, y_val, x_test, y_test = get_one_norm_park(x, y)
+
+    return (
+        x_train,
+        y_train,
+        x_val,
+        y_val,
+        x_test,
+        y_test,
+    )
+
+
+def get_one_cross_park(
+    x: np.ndarray, y: np.ndarray, folds_num: int
+) -> tuple[np.ndarray, np.ndarray, np.ndarray, np.ndarray, np.ndarray, np.ndarray]:
```

Comment:
Could the return type be changed to `torch.Tensor().float()`, such that we don't have to do that every time we call this function in a notebook?

Reply:
![snip2](https://github.com/user-attachments/assets/0f49be89-f929-47a1-8636-880e90687fd1)